Example/GiniVision/SelectAPIViewController.swift

```diff
@@ -22,16 +22,25 @@ class SelectAPIViewController: UIViewController {
         return ProcessInfo.processInfo.arguments.contains("--UITest")
     }
     
-    var analysisDelegate: AnalysisDelegate?
+    var analysisDelegate: AnalysisDelegate? {
```

Comment:
What? Why would setting the delegate result in presenting a screen? It's a side effect.

Reply:
It made sense before changing the didReview method, but now it is only changed in one place so it doesn't make sense anymore. 👍